lib/analytics_event.rb

```diff
@@ -13,7 +13,7 @@ module Action
     TWITTER_EXIT = 'Twitter Exit'
     WORDPRESS_EXIT = 'WordPress Exit'
     WIDGET_VOTE = 'Vote'
-    WIDGET_SIGNIN = 'Sign in to vote'
+    WIDGET_SIGNIN = 'Sign in to track'
```

Comment:
better/more accurate summary of click intention - from https://github.com/mysociety/alaveteli/pull/3160#discussion_r61054212